tests/py/test_communities.py

```diff
@@ -1,6 +1,9 @@
 import json
 
-from liberapay.exceptions import AuthRequired
+from liberapay.exceptions import (
+    AuthRequired,
+    CommunityAlreadyExists,
+)
```

Comment:
There's no need to split this import into multiple lines, `from liberapay.exceptions import AuthRequired, CommunityAlreadyExists` is less than 80 chars long.

Reply:
Fixed here: 2a96f25164ea687f67ee6a9915a485e9d3950f7b